packages/utils/plugins-helpers/src/types.ts

```diff
@@ -183,6 +183,10 @@ export namespace Types {
     'apollo-engine': ApolloEngineOptions;
   }
 
+  export interface GitHubSchemaOptions {
+    [githubProtocol: string]: { token: string };
```

Comment:
`github:${string}` Do you think this works better?

Reply:
I like that 👍 However, looks like the `website` GH actions can only generate json schema for `number` or `string` index.

Example: https://github.com/dotansimha/graphql-code-generator/actions/runs/4784244922/jobs/8505729237?pr=9332

Reply:
Hi @ardatan @saihaj, should we leave this as `string` until `typescript-json-schema` is fixed upstream?